features/SHA_Extractor/sha_extractor.feature

```diff
@@ -1,104 +1,103 @@
 Feature: SHA Extractor
 
 
-  Scenario: Check that SHA exctractor service has all the information and interfaces it needs to work properly
+  Scenario: Check that SHA extractor service has all the information and interfaces it needs to work properly
     Given SHA extractor service is not started
       And Kafka broker is started on host and port specified in configuration
       And Kafka topic specified in configuration variable "incoming_topic" is created
       And Kafka topic specified in configuration variable "dead_letter_queue_topic" is created
-      And Kafka topic specified in configuration variable "archive_results" is created
+      And Kafka topic specified in configuration variable "outgoing_topic" is created
      When SHA extractor service is started
      Then SHA extractor service does not exit with an error code
       And SHA extractor service should be registered to topic "incoming_topic"
 
 
   Scenario: Check if SHA extractor is able to consume messages from Kafka
-    Given SHA extractor service is started
-      And Kafka broker is started
+    Given Kafka broker is started on host and port specified in configuration
       And Kafka topic specified in configuration variable "incoming_topic" is created
       And Kafka topic specified in configuration variable "dead_letter_queue_topic" is created
-      And Kafka topic specified in configuration variable "archive_results" is created
-     When an event about new tarball being sent by Insights Operator is produced into "incoming_topic" topic
+      And Kafka topic specified in configuration variable "outgoing_topic" is created
+     When an event is produced into "incoming_topic" topic
```

Comment:
because it uses some "wired" data that won't be compatible with other topics/services

Reply:
because "an event" now means one specific file usable just for one topic and won't be compatible with data consumed by aggregator/notification service/whatever.